WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -168,32 +184,22 @@ private void insertLabel(int i)
 
         private void addMapObjectName(MapObject mapObject, BrushConverter bc)
         {
-            name.Text = mapObject.name.Text;
-
+            name.Text = mapObject.Name;
             name.Background = (Brush)bc.ConvertFrom("#FFC6F5F8");
-
             name.FontSize = 18;
-
             name.FontWeight = FontWeights.Bold;
-
             name.Foreground = new SolidColorBrush(Colors.Black);
-
             name.VerticalAlignment = VerticalAlignment.Center;
-
             name.HorizontalAlignment = HorizontalAlignment.Center;
-
             Grid.SetRow(name, 1);
-
             Grid.SetColumnSpan(name, 2);
             Grid.SetColumn(name, 1);
-
             DynamicGrid.Children.Add(name);
         }
 
         private void createRows(string[] contentRows)
         {
             createOneRow(2);
-
             for (int i = 0; i <= contentRows.Length + 1; i++)
             {
                 createOneRow(50);
```

Comment:
Define constant values

Reply:
I defined all constants that are used in multiple places.